test/e2e/storage/pd.go

```diff
@@ -451,29 +453,33 @@ func countReadyNodes(c clientset.Interface, hostName types.NodeName) int {
 }
 
 func verifyPDContentsViaContainer(f *framework.Framework, podName, containerName string, fileAndContentToVerify map[string]string) {
+	var err error
+	var v string
+	errs := []error{}
 	for filePath, expectedContents := range fileAndContentToVerify {
-		var value string
-		// Add a retry to avoid temporal failure in reading the content
-		for i := 0; i < maxReadRetry; i++ {
-			v, err := f.ReadFileViaContainer(podName, containerName, filePath)
-			value = v
+		// Add a timed retry to avoid temporal failure in reading the content
+		err = nil
+		for start := time.Now(); time.Since(start) < time.Minute; time.Sleep(10 * time.Second) {
```

Comment:
If the tests actually wrote data with O_SYNC, then theoretically we shouldn't even need to retry reading the file?

Reply:
changed it so that we actually wait for pod 0 to go down first